doctor/tasks.py

```diff
@@ -621,3 +627,29 @@ def get_document_number_from_pdf(path: str) -> str:
         return ""
     document_number = [dn for dn in document_number_matches[0] if dn]
     return clean_document_number(document_number[0])
+
+
+def extract_recap_pdf(
+    filepath: str,
+    strip_margin: bool = False,
+) -> tuple[str, bool]:
+    """Extract from RECAP PDF
+
+    :param filepath: The path to the PDF
+    :param strip_margin: Whether to remove 1 inch margin from text extraction
+    :return: A tuple containing the text and a boolean indicating ocr usage
+    """
+    content = ""
+    extracted_by_ocr = False
+    with pdfplumber.open(filepath) as pdf:
+        for page in pdf.pages:
+            page_text = get_page_text(page, strip_margin=strip_margin)
+            if page_needs_ocr(page, page_text):
+                extracted_by_ocr = True
+                page_text = extract_with_ocr(page, strip_margin=strip_margin)
+            if "e" not in page_text:
+                # It's a corrupt PDF from ca9. Fix it.
+                page_text = fix_mojibake(page_text)
```

Comment:
Did you figure out of this was still needed?

Reply:
im not sure if we still need this - I assume its still something that can occur occasionally in ca9 - the only way I can think about figuring this out is by processing every ca9 we have scraped recently

Reply:
I don't have a lot of confidence in it. I'd say we should yank it and see what trouble we run into.